Src/Particle/AMReX_ParticleContainerBase.cpp

```diff
@@ -83,6 +84,7 @@ void ParticleContainerBase::SetParticleDistributionMap (int lev, const Distribut
                           m_gdb->ParticleBoxArray(), m_gdb->refRatio());
     m_gdb = &m_gdb_object;
     m_gdb->SetParticleDistributionMap(lev, new_dmap);
+    RedefineDummyMF(lev);
 }
 
 void ParticleContainerBase::SetParticleGeometry (int lev, const Geometry& new_geom)
```

Comment:
Here as well? Looks so similar...

Reply:
Yeah. I had the same question and learned that it is not needed here because the geometry does not affect the iterator